consensus/hotstuff/timestamp/noop.go

```diff
@@ -0,0 +1,19 @@
+package timestamp
+
+import "time"
+
+// NoopBlockTimestamp implements an always valid behavior for BlockTimestamp interface.
+// Can be used by nodes that don't perform validation of block timestamps.
+type NoopBlockTimestamp struct{}
+
+func NewNoopBlockTimestamp() *NoopBlockTimestamp {
+	return &NoopBlockTimestamp{}
+}
+
+func (n NoopBlockTimestamp) Build(time.Time) time.Time {
+	return time.Time{}
```

Comment:
⚠️ 
Even if we don't impose any constraints on the block time, I think we should still _report an accurate time_. Otherwise, we essentially set the time stamps for collections to a dummy value.  

```suggestion
	return time.Now().UTC()
```